README.md

```diff
@@ -51,11 +51,14 @@
 ## Table of Contents
 
 - [What makes Saleor special?](#what-makes-saleor-special)
+- [Why API-only Architecture?](#why-api-only-architecture)
 - [Features](#features)
 - [Installation](#installation)
 - [Documentation](#documentation)
+- [Saleor Platform](#saleor-platform)
+- [Storefront](#storefront)
```

Comment:
Not sure if we want to mention the repos that we are actually not supported

Reply:
I've just updated TOC of our README - we have it already at https://github.com/saleor/saleor#storefront 😄